foxy/ci-nightly-cross-vendor-connext-cyclonedds.yaml

```diff
@@ -0,0 +1,96 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  NDDSHOME: '/opt/rti.com/rti_connext_dds-5.3.1'
+  RMW_IMPLEMENTATIONS: 'rmw_connext_cpp:rmw_cyclonedds_cpp'
+  ROS_PYTHON_VERSION: '3'
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+build_tool: colcon
+build_tool_args: '--cmake-args -DCMAKE_BUILD_TYPE=Release -DSKIP_SINGLE_RMW_TESTS=1 --no-warn-unused-cli'
+install_packages:
+- default-jdk # for CycloneDDS
+- maven # for CycloneDDS
+jenkins_job_label: ci-agent
+jenkins_job_priority: 47
+jenkins_job_timeout: 300
+jenkins_job_upstream_triggers:
+- nightly-release
+package_selection_args: '--paths src/ros2/system_tests/test_communication'
```

Comment:
This approach doesn't work since the `ros_buildfarm` always passes `--base-paths` in addition: see https://github.com/ros-infrastructure/ros_buildfarm/blob/7175f9212072b92f540d01b0b2bcf4d5bc31c9e3/ros_buildfarm/colcon.py#L22.

Instead passing `--packages-select test_communication` which works because all not selected packages will be marked with a `COLCON_IGNORE` file: see #79.